developers/developers.md

```diff
@@ -2,9 +2,13 @@
 
 ## How to Get the Code
 
-ArduSub is hosted and maintained on [github](https://github.com/ardupilot/ardupilot.git). You can clone the repository or download the source code as a zip file from [github](https://github.com/ardupilot/ardupilot.git).
+ArduSub is hosted and maintained on [github](https://github.com/ardupilot/ardupilot.git). You need to clone the repository to have full code access including submodules.
```

Comment:
are submodules not included in the zip?!

Reply:
never was, that is a know GitHub behaviour.

![screenshot from 2018-05-30 15-51-27](https://user-images.githubusercontent.com/1215497/40741449-f54b3402-6421-11e8-8cc5-aba9d1da0306.png)